src/platform/forms-system/src/js/components/SchemaForm.jsx

```diff
@@ -89,8 +89,8 @@ class SchemaForm extends React.Component {
     return true;
   }
 
-  onError() {
-    const formContext = set('submitted', true, this.state.formContext);
+  onError(hasSubmitted = true) {
+    const formContext = set('submitted', hasSubmitted, this.state.formContext);
```

Comment:
Can you add tests for this that captures what this change represents? It looks like we have a test for the current behavior already:

https://github.com/department-of-veterans-affairs/vets-website/blob/e3603f2940f68f536d8d529bc1cec66cb90e9788/src/platform/forms-system/test/js/components/SchemaForm.unit.spec.jsx#L115-L119

Reply:
hey @bkjohnson added a new test to verify the val of submitted is false when `onError(false)` is called... thanks for checking it out 👍